web-ui/src/main/resources/catalog/components/metadataactions/partials/related.html

```diff
@@ -174,6 +174,7 @@ <h3 data-ng-if="::(r.title | gnLocalized: lang).length">{{::(r.title | gnLocaliz
                 {{ (r.description | gnLocalized: lang | striptags)}}</a>
               </span>
               <span ng-bind-html="r.url | gnLocalized: lang | linky:'_blank'"></span>
+              <span gn-copy-to-clipboard="{{r.url | gnLocalized: lang}}" gn-copy-button-only="false"></span>
```

Comment:
In this case, as the url is displayed as a link previously, would not make sense to show only the button?